products/cloudasset/api.yaml

```diff
@@ -105,6 +105,39 @@ objects:
                 required: true
                 description: |
                   Destination on Cloud Pubsub topic.
+      - !ruby/object:Api::Type::NestedObject
+        name: condition
+        required: false
```

Comment:
This defaults to false, so prefer it to be unspecified

Reply:
Removed 'required' attribute